css/styles.css

```diff
@@ -0,0 +1,82 @@
+.body {
+	padding: 30 200px;
+
+}
+
+h1 {
+	font-family: Futura;
+	display: block;
+  text-align: center;
+  color:black;
+  font-size: 24pt;
+  text-decoration: none;
+}
+
+h2 {
+  font-family: Futura;
+  display: block;
+  text-align: center;
+  color:black;
+  font-size: 14pt;
+  text-decoration: none;
+}
+
+.avatar {
+	height: 130px;
+	display: block;
+  margin-left: auto;
+  margin-right: auto;
+  padding-bottom: 0px;
+}
+
+a:link {
+	text-decoration: none;
+}
+
+.nav{
+    list-style:none;
+    margin:0;
+    padding:0;
+    text-align:center;
+    font-family: Futura;
+}
+.nav li{
+    display:inline;
+}
+.nav a{
+    display:inline-block;
+    padding:10px;
+}
+
+.aboutme {
+	font-family: Baskerville;
+	text-align: center;
+	padding-left: 200px;
+	padding-right: 200px;
+
+}
+
+.albums {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.albums > img {
+  width: 80px;
+  padding: 8px;
+}
+
+.content {
+  padding-top: 15px;
+}
+
+.contactsub {
+  font-size: 8px;
+  text-align: center;
```

Comment:
Since you're applying `text-align: center` to multiple elements. It would be best to apply it to a root elelment that contains the elements you you want styled that way.